roles/confluent.kafka_broker/tasks/dynamic_groups.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Gather OS Facts
+  setup:
+    filter: ansible_os_family
+    gather_subset:
+      - '!all'
```

Comment:
```suggestion
```
Is the `gather_subset` really required when a `filter` is specified? I am not an expert on the setup module, but the documentation indicates that `filter` should be enough.

Reply:
I did some rudimentary timestamp test and did not see any difference in performance with/without the gather_subset. so ill remove it. nice catch. There are other examples of the setup module getting used in the code but probably wont make that change in this PR